examples/next/docs/js/demo/src/utils.js

```diff
@@ -0,0 +1,39 @@
+import { data } from './constants';
+
+const randomName = () =>
+  ["عمر", "علي", "عبد الله", "معتصم"][Math.floor(Math.random() * 3)];
+const randomCountry = () =>
+  ["تركيا", "مصر", "لبنان", "العراق"][Math.floor(Math.random() * 3)];
+const randomDate = () =>
+  new Date(Math.floor(Math.random() * Date.now())).toLocaleDateString('pl-pl').replace(/\./g, '/')
```

Comment:
It is not crucial to have correct dates at this time, but I think it would be slightly more elegant to use this, with the same result:

`new Date(Math.floor(Math.random() * Date.now())).toLocaleDateString('en-gb')`

Reply:
I've fixed it within ad1de94103952ae6f18aa25948a7d8633422ed8f.